code/parachain/runtime/picasso/src/ibc.rs

```diff
@@ -134,18 +134,32 @@ impl pallet_ibc::Config for Runtime {
 	type ParaId = parachain_info::Pallet<Runtime>;
 	type RelayChain = RelayChainId;
 	type WeightInfo = weights::ibc::WeightInfo<Self>;
-	type AdminOrigin = EnsureRootOrOneThirdNativeTechnical;
-	type FreezeOrigin = EnsureRootOrOneThirdNativeTechnical;
 	type SpamProtectionDeposit = SpamProtectionDeposit;
 	type IbcAccountId = Self::AccountId;
-	type TransferOrigin = EitherOf<
-		EnsureSignedBy<ReleaseMembership, Self::IbcAccountId>,
-		EnsureSignedBy<TechnicalCommitteeMembership, Self::IbcAccountId>,
-	>;
-	type RelayerOrigin = EnsureSignedBy<TechnicalCommitteeMembership, Self::IbcAccountId>;
 	type HandleMemo = ();
 	type MemoMessage = MemoMessage;
 	type Ics20RateLimiter = ConstantAny;
 	type IsReceiveEnabled = ConstBool<true>;
 	type IsSendEnabled = ConstBool<true>;
+
+	#[cfg(feature = "testnet")]
+	type AdminOrigin = EnsureRootOrOneThirdNativeTechnical;
+	#[cfg(feature = "testnet")]
+	type FreezeOrigin = EnsureRootOrOneThirdNativeTechnical;
+	#[cfg(feature = "testnet")]
+	type TransferOrigin = system::EnsureSigned<Self::IbcAccountId>;
+	#[cfg(feature = "testnet")]
+	type RelayerOrigin = system::EnsureSigned<Self::IbcAccountId>;
+
+	#[cfg(not(feature = "testnet"))]
+	type AdminOrigin = EnsureRootOrOneThirdNativeTechnical;
```

Comment:
This should be 2/3 no?

Reply:
agreed

Reply:
seems like it is correct, but if with or without testfast feature has the the same AdminOrigin and FreezeOrigin, maybe we shouldnt put them separately